portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <div id="content">
       <h1>Stefano's Portfolio</h1>
-      <img src="/images/selfie.jpg" alt="Stefano's selfie" width="250">
+      <img src="images/selfie.jpg" alt="Stefano's selfie" width="250">
```

Comment:
this change is actually interesting. I usually would recommend always using absolute paths and not relative ones.
See e.g. https://www.keycdn.com/blog/relative-path which also argues for absolute paths. It also includes an interesting mention of relative protocol URLs which are good to know about.

Reply:
I wasn't aware of this benefits, thanks.